crane-cloud/backend
# implement activity logs changes

# Description

Added additional changes to activity logs

## Type of change

Please delete options that are not relevant.



## Trello Ticket ID

https://trello.com/c/TKwkNJ4p/1311-user-activity-logs-changes

## How Can This Be Tested?

Input project id, app id or database id for activity logs. User should be able to see all activities for that project, app or database done by other users as well. Non-member users cannot see logs for a project. The logs should now also include the user email and user name.


## Checklist:
